### PR TITLE
KAW-7695 Add table block

### PR DESCRIPTION
**Pull request by TomaszDziezykNetcentric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #KAW-7695

Test URLs:
- Before: https://main--edge-delivery-bimota--netcentric.hlx.page/tools/sidekick/blocks/table
- After: https://kaw-7695-table--edge-delivery-bimota--netcentric.hlx.page/tools/sidekick/blocks/table

### CODE REVIEW

File: blocks/table/table.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.table tr:first-of-type td {
+  border-top: none;
+}
+
+.table td p {
+  margin: 0;
+}
+
+.table tr:nth-child(2n) td {
+  background-color: #fff;
```
Comment (lakshmishriaswathnarayana) [inline blocks/table/table.css]: can you use variable here?

**Review by lakshmishriaswathnarayana — CHANGES_REQUESTED** [review]

**Review by lakshmishriaswathnarayana — CHANGES_REQUESTED** [review]
Since you have handle this component using table, can you also handle [this](https://bimota.it/cookie-policy/) as variant

File: blocks/table/table.css (hunk 1 above)
Comment (TomaszDziezykNetcentric) [reply blocks/table/table.css]: done

**Comment by TomaszDziezykNetcentric** [comment]
> Since you have handle this component using table, can you also handle [this](https://bimota.it/cookie-policy/) as variant

done

**Review by lakshmishriaswathnarayana — APPROVED** [review]

**Pull request merged**